### DISCUSSION TITLE
[docs] Add to breaking changes that the `loop` option was renamed to the `loopFocus` option.

### DISCUSSION THREAD

**Discussion by anneau** [chakra-ui/ark#2495] [General]
## Overview

[In this pull request](https://github.com/chakra-ui/ark/pull/2419/files), the `loop` option is renamed to the `loopFocus` option, but is not listed in the breaking changes, which is confusing.

**Comment by cschroeter** [comment]
@anneau 

Thanks mate. I've updated the change log.
